File: Java Course/Basics/palindrome.java
import java.util.Scanner;

public class palindrome {
    public static void main(String[] args) {
        // palindrome
        Scanner input = new Scanner(System.in);
        System.out.print("Enter the word : "); //madam
        String name = input.next();
        String rev = "";
        for(int i = name.length()-1;i>=0;i--){
            rev+=name.charAt(i);
        }
        
        if(name.equals(rev)){
            System.out.println("It is palindrome");
        }
        else{
            System.out.println("It is not palindrome");
        }

        input.close();
    }
}

File: Python Course/Problems/palindrome.py
  # palindrome
word = input("Enter the word : ")
rev = word[::-1]
# arr = list(word)
# rev = ""
# for i in range (len(arr)-1,-1,-1):
#     rev += arr[i]
if (rev==word):
    print("It is palindrome")
else:
    print("It is not palindrome")